<?xml version="1.0" ?>
<launch>
  <arg name="cmd" default="move_up" />
  <group ns="robot1">
    <node name="move" pkg="chair_manipulation_grasp_planning" type="move_node" output="screen">
      <param name="cmd" type="string" value="$(arg cmd)" />
      <param name="prefix" value="robot1" />
      <param name="gripper_group" value="robot1_gripper" />
    </node>
  </group>
  <group ns="robot2">
    <node name="move" pkg="chair_manipulation_grasp_planning" type="move_node" output="screen">
      <param name="cmd" type="string" value="$(arg cmd)" />
      <param name="prefix" value="robot2" />
    </node>
  </group>
</launch>